OpenEdXMobile/build.gradle

```diff
@@ -420,3 +421,9 @@ screenshots {
     testApkTarget = 'packageProdDebuggableAndroidTest'
     connectedAndroidTestTarget = 'connectedProdDebuggableAndroidTest'
 }
+
+// Google suggests to add this at the bottom
+// https://developers.google.com/android/guides/google-services-plugin
```

Comment:
I guess this doesn't need to be at the absolute bottom, and can just below the dependencies and configurations section. It might be better to move it it relates to a dependency.